grasshopper.json

```
@@ -0,0 +1,37 @@
+{
+    "version": "1.1.4",
+    "description": "The free and easy SSH client",
```

`The free` is stated in license property

```suggestion
    "description": "SSH client",
```

---

The description is from their website. I'm aware it's in the license.

---

What does website have to do with description?

Having same information twice is unnecessary.

---

To quote the reference page:
> A one line string containing a short description of the program. Don’t include the name of the program, if it’s the same as the app’s filename.

This is short, descriptive, and does not repeat the file's name.